docs/site/translation/number-currency-formats/number-symbols.md

```diff
@@ -6,7 +6,9 @@ title: Number Symbols
 
 The following symbols are used to format numbers. The Approved symbols for the locale will be substituted for the placeholders described in [Number Patterns](/translation/number-currency-formats/number-and-currency-patterns). 
 
-For English regional locales (e.g. en\_DE) where English is not the primary language of the country/region (e.g. en\_DE; English as used in Germany), the number formats and date formats should follow the **English formatting usage** in that country/region. Often, the formatting usage in English tend to follow the British or American formatting rather than the formatting of the primary language. Consult with the English versions of prominent magazines or newspapers for guidance on date and number format usage.
+For regional locales (e.g. en\_DE) where the language is not the primary language of the country/region (e.g. en\_DE; English as used in Germany), the number formats and date formats should follow the **formatting usage** as commonly used by the people who live in that country/region. This can often be difficult to determine with languages such as English that are operating as a lingua franca. In these situations, much of the available content is written for an international audience and the majority of the readers using the language are from other countries/regions.
```

Comment:
```suggestion
For regional locales where the language is not the primary language of the country/region (e.g. en\_DE, English as used in Germany; or es_SE, Spanish as used in Sweden), the number formats and date formats should follow the formatting usage **of that language** as commonly used by the people who live in that country/region. This can be difficult to determine with languages that are operating as a lingua franca by people not using it as their first language. In these situations, much of the available content is written for an international audience and most of the readers are from other countries/regions.
```